### ISSUE TITLE
Remove fp_binary dependency from ddc driver

### ISSUE THREAD

**Issue by JennySmith888** [MazinLab/MKIDGen3#26]
`fp_binary` has not proved very useful and should ideally be removed from the requirements to enhance maintainability. Recommend replacing functionality with `np.round`, typecasting to np ints and using bit manipulation.

**Comment by baileyji** [comment]
I vaguely recall us using it in a few core cases early on to simulate fixed point math. Before we need to make a list of all code that uses it and decide what do do with it, maybe hiving that off into mkidanalysis or some other optional package. I think removing support from the drivers and any FRS paths would be relatively straightforward. That would open a third door so that it becomes an "optional" package that is only required for the python implementation of the fixed point opfb/fir. Please do not assign this to me, I'm not clear it is on a critical path.

**Comment by baileyji** [comment]
Got it. 

Subclasses of what? Are you referring to the older DDC drivers? I think we could axe them and relegate them to either a tagged commit or moving the python to the hls repo. Without specifics examples any refactoring I do might not address what you see as a source of confusion. 

I'll look at pulling it out of all the current code and leaving it as an optional needed for the software implementation of opfb. I think that shouldn't be too hard and is a reasonable goal that doesn't lose valuable history.

**Comment by JennySmith888** [comment]
Yes the older DDC drivers. I don't think the DDC driver needs to do any subclassing anymore. I think your proposal of tagged commit makes the most sense. That plus eliminating the `fp_binary` dependence is all the refactoring I am proposing.
